docs/CONTRIBUTING.md

```diff
@@ -48,3 +50,65 @@ Every pull request should have a description that explains *why* the change is b
 The description adds context that is critical for reviewers when giving feedback.
 
 For more tips, see GitHub's blog entry on [how to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request).
+
+## Editing Docs Locally
+
+If you want to submit a pull request to update the docs, you will need to [make a fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this repo and clone it.
+
+Currently all CircleCI docs are located in `circleci-docs/jekyll/_cci2`.
+
+1. Create a branch and switch to it:
+
+    `git checkout -b <branch-name>`
+
+2. Add or modify Asciidoc and Markdown files in these directories according to our [style guide](https://circleci.com/docs/style/style-guide-overview).
+
+3. When you are happy with your changes, commit them with a message summarizing what you did:
+
+    `git commit -am "commit message"`
+
+4. Push your branch up:
+
+    `git push origin <branch-name>`
+
+## Adding New Articles
+
+New articles can be added to the [jekyll/_cci2](https://github.com/circleci/circleci-docs/tree/master/jekyll/_cci2) directory in this repo.
+
+We are slowly converting all oue content over to the Asciidoc format. So all new pages should be submitted in Asciidoc.
+
+When you make a new article, you will need to add some [**document attributes**](https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes/) and [**front matter**](https://jekyllrb.com/docs/frontmatter/). These contains metadata about the article you are writing and are required so everything works on our site.
+
+Document attributes and Front matter for our docs will look something like:
+
+```
+---
+version:
+- Server v4.x
+- Admin
+---
+= Page title
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A short description of the content on the page.
+```
+
+`page-layout` and the page title are the only required variables. `page-layout` associates an HTML layout with the page, `classic-docs` is used across our docs. `title` will appear at the top of your article.
+
+## Docker Tag List for CircleCI Convenience Images
+
+The Docker tag list for convenience images, located in ./jekyll/_cci2/circleci-images.md, is dynamically updated during a CircleCI build.
+There's usually no need to touch this.
+If you would like to see an updated list generated locally however, you can do so by running `./scripts/pull-docker-image-tags.sh` from the root of this repo.
+Note that you will need the command-line tool [jq](https://stedolan.github.io/jq/) installed.
+
+## Updating `browserlist-stats.json`
```

Comment:
Is this still relevant? This seems like something that my live with docs-platform now.

Reply:
Good question. @rsylvian I think we can remove this section from our contributing guide but just want to confirm.

Reply:
Correct! :) Thanks for the cleanup!